lib/api/model/model.dart

```diff
@@ -213,6 +213,28 @@ class RealmEmojiItem {
   Map<String, dynamic> toJson() => _$RealmEmojiItemToJson(this);
 }
 
+/// As in [InitialSnapshot.realmVideoChatProvider].
+///
+/// For docs, search for "realm_video_chat_provider:"
+/// in <https://zulip.com/api/register-queue>.
+@JsonEnum(fieldRename: FieldRename.snake, valueField: "apiValue")
+enum RealmVideoChatProvider {
+  none(apiValue: 0),
+  jitsi(apiValue: 1),
+  zoomUser(apiValue: 3),
+  bigBlueButton(apiValue: 4),
+  zoomServer(apiValue: 5),
```

Comment:
In some of these names, let's match the API documentation a bit more: `jitsiMeet`, `zoomUserOAuth`, `zoomServerToServerOAuth`.